.gitignore

```diff
@@ -37,3 +37,4 @@ cmd/opentelemetry/cmd/agent/opentelemetry-agent-*
 cmd/opentelemetry/cmd/ingester/opentelemetry-ingester-*
 cmd/opentelemetry/cmd/all-in-one/opentelemetry-all-in-one-*
 __pycache__
+.DS_Store
```

Comment:
I would prefer this be added to your local `~/.gitexcludes` file as there should not be a reason to commit this file into a repository, and it's MacOS specific.

Reply:
> I would prefer this be added to your local `~/.gitexcludes` file as there should not be a reason to commit this file into a repository, and it's MacOS specific.

added DS_Store to my .gitexcludes file